x-pack/legacy/plugins/siem/public/components/inspect/translations.ts

```diff
@@ -13,3 +13,43 @@ export const INSPECT = i18n.translate('xpack.siem.inspectDescription', {
 export const CLOSE = i18n.translate('xpack.siem.inspect.modal.closeTitle', {
   defaultMessage: 'Close',
 });
+
+export const SOMETHING_WENT_WRONG = i18n.translate(
+  'xpack.siem.inspect.modal.somethingWentWrongDescription',
+  {
+    defaultMessage: 'Sorry about that, something went wrong.',
+  }
+);
+export const INDEX_PATTERN = i18n.translate('xpack.siem.inspect.modal.indexPatternLabel', {
+  defaultMessage: 'Index pattern',
+});
+
+export const INDEX_PATTERN_DESC = i18n.translate(
+  'xpack.siem.inspect.modal.indexPatternDescription',
+  {
+    defaultMessage: 'The index pattern that connected to the Elasticsearch indices.',
```

Comment:
Would it make sense to include a gentle hint on how to change this value, perhaps with a description like the following?

`The indices searched by this query, as configured in Kibana > Advanced Settings`